src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Resident.kt

```diff
@@ -31,7 +31,8 @@ data class Resident(
 
   val dateOfBirth: LocalDate?,
 
-  val age: Int?,
+  @NotNull
```

Comment:
is this nullable for the main resident?

Reply:
Ahh maybe not, I thought there must be a reason I made it nullable before. I was just looking at prototype and it looked like it was not optional, but nothing says that so I'll make it nullable again